utils/name-check.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * This function determines whether a string contains the characters of a specified string.
+ * @param { String } name 
+ * @returns { Boolean }
+ */
+const nameCheck = (name) => {
+    const badCharacter = "`~!@#$%^&*()_+-=/*-/.,<>?';:|";
+    for (let i = 0; i < name.length; i++) {
+        for (let k = 0; k < badCharacter.length; k++) {
+            if (name.charAt(i) === badCharacter.charAt(k)) {
```

Comment:
I didn't understand how this is working

Reply:
This function checks if the username contains any of the characters in badCharacter string
Example: 
nameCheck("sira$j") returns false
nameCheck("Chris") returns true